src/aggregate.ts

```diff
@@ -121,14 +144,32 @@ abstract class AggregateField {
     return new Count();
   }
 
+  /**
+   * Gets a copy of the Sum aggregate field.
+   *
+   * @returns {Sum}
+   */
+  static sum(property: string): Sum {
+    return new Sum(property);
+  }
+
+  /**
+   * Gets a copy of the Average aggregate field.
+   *
+   * @returns {Average}
+   */
+  static average(property: string): Average {
+    return new Average(property);
+  }
+
   /**
    * Gets a copy of the Count aggregate field.
```

Comment:
should this be updated to remove "Count" since it applies to all?

Reply:
Done